studio/.env.local.example

```
@@ -0,0 +1,3 @@
+SANITY_STUDIO_PROJECT_ID="projectId"
```

Following the same format as the `studio` dir.
```suggestion
SANITY_STUDIO_PROJECT_ID="<paste your project ID here>"
```